docs/verify/stabilizer-circuit-simulation.ipynb

```diff
@@ -0,0 +1,199 @@
+    "\n",
+    "Stabilizer circuits are an important restricted class of quantum circuits that can be efficiently simulated classically. There are several equivalent ways to define stabilizer circuits. One way to define them is that a stabilizer circuit is a circuit that consists solely of the following gates:\n",
+    "\n",
+    "- [CX](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.CXGate)\n",
+    "- [Hadamard](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.HGate)\n",
+    "- [S](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.SGate)\n",
+    "- [Measurement](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.Measure)\n",
+    "\n",
+    "Note that using Hadamard and S, we can construct any Pauli rotation gate ([$R_x$](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.RXGate), [$R_y$](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.RYGate), and [$R_z$](https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.circuit.library.RZGate)) that has an angle contained in the set $\\{0, \\frac{\\pi}{2}, \\pi, \\frac{3\\pi}{2}\\}$ (up to global phase), so we can include these gates in the definition as well. As a result, if the parameters of the EfficientSU2 circuit are restricted to these values, it can be efficiently simulated!\n",
+    "\n",
+    "In the following cell, we run the circuit with the Sampler primitive backed by the stabilizer circuit simulator, using parameters chosen randomly such that the circuit is guaranteed to be a stabilizer circuit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_aer.primitives import Sampler\n",
+    "import numpy as np\n",
+    "\n",
+    "measured_circuit = circuit.copy()\n",
+    "measured_circuit.measure_all()\n",
+    "\n",
+    "rng = np.random.default_rng(1234)\n",
+    "params = rng.choice(\n",
+    "    [0, np.pi / 2, np.pi, 3 * np.pi / 2],\n",
+    "    size=circuit.num_parameters,\n",
+    ")\n",
+    "\n",
+    "# Initialize a Sampler backed by the stabilizer circuit simulator\n",
+    "exact_sampler = Sampler(backend_options=dict(method=\"stabilizer\"))\n",
+    "job = exact_sampler.run(measured_circuit, params)\n",
+    "exact_quasis = job.result().quasi_dists[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The stabilizer circuit simulator also supports noisy simulation, but only for a restricted class of noise models. Specifically, the noise must be characterized by a [Pauli error](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.noise.pauli_error.html#qiskit_aer.noise.pauli_error) channel. [Depolarizing error](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.noise.depolarizing_error.html) falls into this category, so it can be simulated too.\n",
```

Comment:
I think that you can also simulate a `ReadoutError` using the stabilizer simulator

Reply:
Thanks, I updated the example to use readout error as well.